### PR TITLE
feat (MatchHistory) add beginTime / endTime option

### PR DESCRIPTION
**Pull request by Colorfulstan**
add beginTime / endTime to request for getMatchHistory()

### CODE REVIEW

File: lib/lolapi.js [hunk 1]
```diff
@@ -148,7 +148,14 @@
             if (options.endIndex) {
                 historyOptions += '&endIndex=' + options.endIndex;
             }
+            if (options.beginTime) {
```
Comment (claudiowilson) [inline lib/lolapi.js]: Can you make the check look for undefined? If someone used a 0 begin time for example this would return false.

Comment (claudiowilson) [inline lib/lolapi.js]: Type coercion lol

Comment (Colorfulstan) [inline lib/lolapi.js]: since beginTime 0 would result in the same response as if the option is not set i don't see the point of the suggested check

Comment (claudiowilson) [inline lib/lolapi.js]: Tested and that does seem to be the case, cool

**Pull request merged**